sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/samples/Generated/Samples/Sample_ConnectedClusterBgpPeerCollection.cs

```diff
@@ -21,7 +21,7 @@ public partial class Sample_ConnectedClusterBgpPeerCollection
         public async Task CreateOrUpdate_BgpPeersCreateOrUpdate()
         {
             // Generated from example definition: 2024-03-01/BgpPeers_CreateOrUpdate.json
-            // this example is just showing the usage of "BgpPeer_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "BgpPeers_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/kubernetesruntime/KubernetesRuntime.Management/bgppeer.tsp#L41